src/main/java/in/projecteka/consentmanager/common/heartbeat/Heartbeat.java

```diff
@@ -52,6 +54,8 @@
     }
 
     private boolean isRedisUp() throws IOException {
+        if (cacheMethodProperty.getMethodName().equals(CACHE_METHOD_NAME))
+            return true;
```

Comment:
method says RedisUp but checking other cache type inside this method does not look good.

isCacheUp()

        if isRedisCache => return isRedisUp() otherwise return true

Reply:
doCaseInsensitiveEquality when it makes sense.

Reply:
done